corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -180,6 +188,11 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         var mapping = {
             caption: {
                 update: function (options) {
+                    let parentForm = getParentForm(self);
+                    let oqps = parentForm.displayOptions.oneQuestionPerScreen != undefined && parentForm.displayOptions.oneQuestionPerScreen();
```

Comment:
nit: rename `oqps` var or get rid of the variable by rearranging so that
```
if (parentForm.displayOptions.oneQuestionPerScreen === undefined || !parentForm.displayOptions.oneQuestionPerScreen()) {
    return options.data ? DOMPurify.sanitize(options.data.replace(/\n/g, '<br/>')) : null;
} else {
    return null
}
```

Reply:
that's fair. I'm changing it to `let oneQuestionPerScreen`.  I've updated this logic where the `if` check has additional statement so I thought it would be a little more clear to have this var.